Gems/Atom/Tools/MaterialEditor/Code/Source/Platform/Linux/MaterialEditor_Linux.cpp

```diff
@@ -6,33 +6,6 @@
  *
  */
 
-#include <AzCore/std/string/string.h>
-#include <QWidget>
-#include <AzFramework/Windowing/WindowBus.h>
-
 namespace Platform
```

Comment:
We should delete the three `MaterialEditor_<platform>.cpp` files as they are not needed anymore